index.html

```diff
@@ -6163,6 +6163,9 @@ <h2>Thing Model Declaration</h2>
         <a>Thing Model</a> definitions MUST use the keyword <em>@type</em> at top level 
         and a value of type string or array that equals or respectively contains <code>tm:ThingModel</code>.
     </span>
+    <span class="rfc2119-assertion" id="tm-context-requirement">
+        Additionally, in order to identify it as a JSON-LD document, <a>Thing Model</a> definitions MUST use the keyword <em>@context</em> at top level with same rules as a Thing Description.
```

Comment:
Minor: you might want to use `<code>@context</code>` instead of `<em>`

Reply:
Done. I had copied it from the above one, now both use `<code>`